build/teamcity/internal/release/build-and-publish-patched-go/diff.patch

```diff
@@ -148,18 +176,20 @@ diff --git a/src/runtime/sizeof_test.go b/src/runtime/sizeof_test.go
 index fb9195481a..55e37287c2 100644
 --- a/src/runtime/sizeof_test.go
 +++ b/src/runtime/sizeof_test.go
-@@ -21,6 +21,6 @@ func TestSizeof(t *testing.T) {
+@@ -21,7 +21,7 @@ func TestSizeof(t *testing.T) {
  		_32bit uintptr // size on 32bit platforms
  		_64bit uintptr // size on 64bit platforms
  	}{
 -		{runtime.G{}, 252, 408},   // g, but exported for testing
 +		{runtime.G{}, 260, 416},   // g, but exported for testing
  		{runtime.Sudog{}, 56, 88}, // sudog, but exported for testing
  	}
-diff -ru a/src/context/context.go b/src/context/context.go
+
```

Comment:
Blank line here seems wrong

Reply:
To be clear: if the diff apply works, don't worry about removing the blank line. Just a thought I had as I was reviewing.